spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/custom/SentinelBeanPostProcessor.java

```diff
@@ -174,7 +173,7 @@ private boolean checkSentinelProtect(RootBeanDefinition beanDefinition,
 	private boolean checkMethodMetadataReadingVisitor(RootBeanDefinition beanDefinition) {
 		return beanDefinition.getSource() instanceof MethodMetadata
 				&& ((MethodMetadata) beanDefinition.getSource())
-						.isAnnotated(SentinelRestTemplate.class.getName());
```

Comment:
You may run `mvn clean install` to execute the checkstyle plugin before you submit a pull request. It seems that you didn't execute the command?